_posts/2018-05-17-running-strimzi-on-openshift-online.md

```diff
@@ -109,13 +109,13 @@ spec:
     type: Recreate
 ```
 
-Once you apply this YAML, Cluster Operator should deploy and start watching for config maps.
+Once you apply this YAML, Cluster Operator should deploy and start watching for ConfigMaps.
```

Comment:
I have not a strong opinion but we have to define a rule. Even in the documentation we are using both ConfigMap and config map. It could be better to use one of them consistently. Whatever your decision will be I'm fine, just use the same :-)

Reply:
I'm fine with ConfigMap!